engine/src/flutter/ci/builders/linux_web_engine_test.json

```diff
@@ -2,6 +2,28 @@
   "_comment": "THIS IS A GENERATED FILE. Do not edit this file directly.",
   "_comment2": "See `generate_builder_json.dart` for the generator code",
   "builds": [
+    {
+      "name": "web_tests/test_bundles/dart2js-canvaskit-experimental-webparagraph",
+      "drone_dimensions": [
+        "device_type=none",
+        "os=Linux"
+      ],
+      "generators": {
+        "tasks": [
+          {
+            "name": "compile bundle dart2js-canvaskit-experimental-webparagraph",
+            "parameters": [
+              "test",
+              "--compile",
+              "--bundle=dart2js-canvaskit-experimental-webparagraph"
+            ],
+            "scripts": [
+              "flutter/lib/web_ui/dev/felt"
+            ]
+          }
+        ]
+      }
+    },
```

Comment:
Nothing should be changed here expect the Mac version: https://github.com/flutter/flutter/pull/177669/files#diff-bcc5fdaded80eb759a2ad42c43ab64d5630608a1a3a4817d32e0fdb508e05ecaR400